src/main/java/com/google/sps/data/OrganizationUpdater.java

```diff
@@ -54,7 +56,7 @@ public void updateOrganization(HttpServletRequest request, GivrUser user, boolea
     Set<String> requiresModerator = new HashSet<String>();
     Map<String, String> properties = new HashMap<String,String>();
     boolean isMaintainer = user.isMaintainer();
-    boolean isModerator = isModerator = user.isModeratorOfAnyOrg();
+    boolean isModerator = user.isModeratorOfAnyOrg();
```

Comment:
I think there's a subtle bug here; this should be if the user is a moderator of the org listed in the request! Can you please leave a comment and send an immediate PR to fix ?

Reply:
yes!